CHANGES.rst

```diff
@@ -4,7 +4,8 @@ z3c.saconfig
 0.15 (unreleased)
 =================
 
-- Nothing changed yet.
+- all tests are ported to python3 compatibility [nazrulworld]
```

Comment:
Would be nice to mention which specific Python 3 versions are supported (AFAICT they're 3.5 and 3.6).

Usually we phrase this as

> - Added Python 3.5 and 3.6 compatibility